eureka-core/src/main/java/com/netflix/eureka/registry/RemoteRegionRegistry.java

```diff
@@ -388,6 +392,7 @@ private Applications fetchRemoteRegistry(boolean delta) {
                 int httpStatus = httpResponse.getStatusCode();
                 if (httpStatus >= 200 && httpStatus < 300) {
                     logger.debug("Got the data successfully : {}", httpStatus);
+                    timeOfLastSuccessfulRemoteFetch = System.currentTimeMillis();
```

Comment:
You also want to update it on line 175 or inside of `private boolean fetchRegistry()` which gets periodically executed.

Reply:
Sorry let me load this in VSCode, this GH PR view is really poor, I misunderstood the place due to collapsed file.

Reply:
OK! I just added the code you suggested, but I'm realizing it ends up in the same place.  Perhaps it makes sense to do it once at the higher level though, rather than at the lower level where I did it originally.

Reply:
OK, viewed this locally, so to me it seems like capturing at the most top level (executor or its core `fetchRegistry`) would be beneficial, as currently this is network-level meaning if the deltas won't match we may update it anyway?

Reply:
100% agree -- just pushed a new version; LMK what you think?

Reply:
Yep, all good 👍

Reply:
Thanks 👍